src/interpreter_funcs/import_obj_mesh.rs

```diff
@@ -64,15 +64,16 @@ impl<C: ObjCache> Func for FuncImportObjMesh<C> {
         let result = self.importer.import_obj(path);
         match result {
             Ok(models) => {
-                // FIXME: @Correctness Join all meshes into one once
-                // we have join implemented for more than just 2
-                // meshes
-
-                let first_model = models.into_iter().next();
-                if let Some(first_model) = first_model {
-                    Ok(Value::Geometry(Arc::new(first_model.geometry)))
-                } else {
+                if models.is_empty() {
                     Err(FuncError::new(FuncImportObjMeshError::Empty))
+                } else {
+                    let geometries: Vec<_> = models
+                        .into_iter()
+                        .map(|model| Arc::new(model.geometry))
+                        .collect();
+
+                    let value = GeometryArrayValue::new(geometries);
+                    Ok(Value::GeometryArray(Arc::new(value)))
```

Comment:
If you want to test it on some real-world files, we've got these: https://onedrive.live.com/?id=A0980D900E318435%211885424&cid=A0980D900E318435
They originally come from students of VSVU, collected by Maros, converted by Kristyna.

Reply:
I'll test, not for this PR. If it doesn't work, it most likely isn't the fault of this the changes introduced here.